File: Examples/CSharp/ProgrammersGuide/WorkingWithProjects/CreateEmptyProjectSaveXML.cs
using System.IO;
using Aspose.Tasks;
using Aspose.Tasks.Saving;

namespace Aspose.Tasks.Examples.CSharp.WorkingWithProjects
{
    public class CreateEmptyProjectSaveXML
    {
        public static void Run()
        {
            //ExStart: CreateEmptyProjectSaveXML
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_Projects();

            Project project = new Project();

            project.Save(dataDir+ "saved.xml", SaveFileFormat.XML);
            //ExEnd: CreateEmptyProjectSaveXML
        }
    }
}

File: Examples/VisualBasic/ProgrammersGuide/WorkingWithProjects/CreateEmptyProjectSaveXML.vb
Imports Microsoft.VisualBasic
Imports System.IO
Imports Aspose.Tasks
Imports Aspose.Tasks.Saving

Namespace Aspose.Tasks.Examples.VisualBasic.WorkingWithProjects
    Public Class CreateEmptyProjectSaveXML
        Public Shared Sub Run()
            'ExStart: CreateEmptyProjectSaveXML
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_Projects()

            Dim project As New Project()

            project.Save(dataDir & "saved.xml", SaveFileFormat.XML)
            'ExEnd: CreateEmptyProjectSaveXML
        End Sub
    End Class
End Namespace